apps/wfo-ui-surf/types/types.ts

```diff
@@ -0,0 +1,210 @@
+
+export enum WfoServiceTicketListTabType {
+    ACTIVE = 'ACTIVE',
+    COMPLETED = 'COMPLETED',
+}
+
+export interface CreateServiceTicketPayload {
+    ims_pw_id: string;
+    jira_ticket_id: string;
+    title_nl: string;
+    type: ServiceTicketType;
+}
+
+export interface OpenServiceTicketPayload {
+    cim_ticket_id: string;
+    title_nl: string;
+    description_nl: string;
+    title_en: string;
+    description_en: string;
+    mail_subject: string;
+}
+
+export interface UpdateServiceTicketPayload extends OpenServiceTicketPayload {
+    start_date?: string;
+    end_date?: string;
+}
+
+export interface CloseServiceTicketPayload extends OpenServiceTicketPayload {
+    end_date?: string;
+}
+
+export enum ServiceTicketProcessState {
+    NEW = 'new',
+    OPEN = 'open',
+    OPEN_RELATED = 'open_related',
+    OPEN_ACCEPTED = 'open_accepted',
+    UPDATED = 'updated',
+    ABORTED = 'aborted',
+    CLOSED = 'closed',
+}
+
+export enum ServiceTicketTransition {
+    RELATING = 'relating',
+    ACCEPTING = 'accepting',
+    ABORTING = 'aborting',
+    OPENING = 'opening',
+    OPEN_AND_CLOSE = 'open_and_close',
+    UPDATING = 'updating',
+    CLOSING = 'closing',
+    CLEANING = 'cleaning',
+}
+
+export type ServiceTicketDefinition = {
+    status_color_field?: string;
+    jira_ticket_id: string;
+    opened_by: string;
+    process_state: ServiceTicketProcessState;
+    start_date: string;
+    create_date: string;
+    last_update_time: string;
+    title_nl: string;
+};
+
+export type ServiceTicketListItem = Pick<
+    Process,
+    | 'workflowName'
+    | 'lastStep'
+    | 'lastStatus'
+    | 'workflowTarget'
+    | 'createdBy'
+    | 'assignee'
+    | 'processId'
+    | 'subscriptions'
+> & {
+    startedAt: Date;
+    lastModifiedAt: Date;
+    productName?: string;
+    productTag?: string;
+    customer: string;
+    customerAbbreviation: string;
+};
+
+export enum ServiceTicketLogType {
+    OPEN = 'open',
+    UPDATE = 'update',
+    CLOSE = 'close',
+}
+
+export interface ServiceTicketLog {
+    entry_time: string;
+    update_nl: string;
+    update_en: string;
+    log_type: ServiceTicketLogType;
+    logged_by: string;
+    transition: ServiceTicketTransition;
+    completed: boolean;
+}
+
+export enum ServiceTicketImpactedObjectImpact {
+    DOWN = 'down',
+    NO_IMPACT = 'no_impact',
+    REDUCED_REDUNDANCY = 'reduced_redundancy',
+    RESILIENCE_LOSS = 'resilience_loss',
+}
+
+export interface ServiceTicketImpactedIMSCircuit {
+    ims_circuit_id: number;
+    ims_circuit_name: string;
+    impact: ServiceTicketImpactedObjectImpact;
+    extra_information?: string;
+}
+
+export interface ServiceTicketCustomer {
+    customer_id: string;
+    customer_name: string;
+    customer_abbrev: string;
+}
+export interface ServiceTicketBackgroundJobCount {
+    number_of_active_jobs: number;
+}
+
+export interface ServiceTicketContact {
+    name: string;
+    email: string;
+    // Todo: add phone?
```

Comment:
For todo's we usuallt refer to a ticket that would resolve the todo eventually (unless another PR in this story takes care of this)